dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
@nmonga91 since we don't yet have a built-in place for exposing the swagger, what do you recommend we do with these files in the meantime? keep them in a working folder somewhere in the repo? stash them somewhere until [DB-395](https://federal-spending-transparency.atlassian.net/browse/DB-395) gets moved into a sprint?